README.md

```diff
@@ -111,6 +111,98 @@ class MyControllerTest extends WebTestCase
 }
 ```
 
+### Methods
+
+#### Check HTTP status codes
+
+##### isSuccessful()
+
+Check that the request succedded:
+
+```php
+$client = static::makeClient();
+$client->request('GET', '/contact');
+
+// Successful HTTP request
+$this->isSuccessful($client->getResponse());
+```
+
+Add `false` as the second argument in order to check that the request failed:
+
+```php
+$client = static::makeClient();
+$client->request('GET', '/error');
+
+// Request returned an error
+$this->isSuccessful($client->getResponse(), false);
+```
+
+In order to test more specific status codes, use `assertStatusCode()`:
+
+##### assertStatusCode()
+
+Check the [HTTP status code](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) from the request:
```

Comment:
I think that the link to wikipedia is not very useful because ```HTTP status code``` is a basic knowledge for a ```Symfony``` developer :-)